Python/Tests/DebuggerTests/AttachTests.cs

```diff
@@ -865,116 +905,285 @@ void main()
+                };
 
-                            await proc.ResumeAsync(TimeoutToken());
-                            attached.SetResult(true);
-                        };
+                await proc.StartListeningAsync();
+                await attached.Task.WithTimeout(20000, "Failed to attach within 20s");
+                await bpHit.Task.WithTimeout(20000, "Failed to hit breakpoint within 20s");
+            });
 
-                        var bpHit = new TaskCompletionSource<bool>();
-                        proc.BreakpointHit += async (sender, args) => {
-                            Console.WriteLine("Breakpoint hit");
-                            bpHit.SetResult(true);
-                            await proc.ResumeAsync(TimeoutToken());
-                        };
+            AssertUtil.ArrayEquals(expectedOutput, actualOutput);
+        }
 
-                        await proc.StartListeningAsync();
-                        await attached.Task.WithTimeout(20000, "Failed to attach within 20s");
-                        await bpHit.Task.WithTimeout(20000, "Failed to hit breakpoint within 20s");
+        private async Task TestPtvsdCommandLine(string args, Uri uri) {
+            if (!HasPtvsdCommandLine) {
+                return;
+            }
 
-                        p.WaitForExit(DefaultWaitForExitTimeout);
-                    } finally {
-                        await DetachProcessAsync(proc);
+            string script = TestData.GetPath(@"TestData\DebuggerProject\AttachPtvsd.py");
+            var expectedOutput = new[] { "stdout", "stderr" };
+            var actualOutput = new List<string>();
+
+            await TestPtvsd("-m ptvsd " + args, script, "", uri, actualOutput, async proc => {
+                var attached = new TaskCompletionSource<bool>(TaskCreationOptions.RunContinuationsAsynchronously);
+                proc.ProcessLoaded += async (sender, e) => {
+                    Console.WriteLine("Process loaded");
+
+                    var bp = proc.AddBreakpoint(script, 11);
+                    await bp.AddAsync(TimeoutToken());
+
+                    await proc.ResumeAsync(TimeoutToken());
+                    attached.SetResult(true);
+                };
+
+                var bpHit = new TaskCompletionSource<bool>(TaskCreationOptions.RunContinuationsAsynchronously);
+                proc.BreakpointHit += async (sender, e) => {
+                    Console.WriteLine("Breakpoint hit");
+                    bpHit.SetResult(true);
+                    await proc.ResumeAsync(TimeoutToken());
+                };
+
+                await proc.StartListeningAsync();
+                await attached.Task.WithTimeout(20000, "Failed to attach within 20s");
+                await bpHit.Task.WithTimeout(20000, "Failed to hit breakpoint within 20s");
+            });
+
+            AssertUtil.ArrayEquals(expectedOutput, actualOutput);
+        }
+
+        [TestMethod, Priority(3)]
```

Comment:
Ah, I didn't realize that's what it is. It was just copied from the existing `ptvsd` test. It has been stable for me so far, but let me do a few runs of this whole batch and verify.